addons/website_sale/models/sale_order.py

```diff
@@ -409,3 +418,38 @@ def _get_shop_warning(self, clear=True):
         if clear:
             self.shop_warning = ''
         return warn
+
+    def _filter_can_send_abandoned_cart_mail(self, website):
```

Comment:
Would suggest not passing the website as argument and ensure that self.website_id is only one record.
```suggestion
    def _filter_can_send_abandoned_cart_mail(self):
        self.website_id.ensure_one()
```